server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -225,7 +228,11 @@ export class RestGitService {
 	public async createRef(params: ICreateRefParamsExternal): Promise<git.IRef> {
 		// We modify this param to prevent writes to external storage if tenant is not linked
 		if (!this.writeToExternalStorage) {
-			params.config.enabled = false;
+			if (params.config === undefined) {
```

Comment:
Nit: can probably be simplified to params.config = { ...params.config, enabled: false };?